examples/app/fourier_animated.py

```diff
@@ -154,7 +152,7 @@ def create_centric_plot(foos, title='', r = 1, y_range=(-2, 2), period = pi/2, c
 cf2 = lambda x: (4*np.cos(3*x))/(3*pi)
 cf3 = lambda x: (4*np.cos(5*x))/(5*pi)
 cf4 = lambda x: (4*np.cos(7*x))/(7*pi)
-fourier = OrderedDict(
+fourier = dict(
```

Comment:
This one might matter, I would double check that the example runs as expected. Or just leave it as-is for now, I am hoping to find time to try and pare down/simplify both fourier examples separately.